config/endpoints.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+return [
+    'plain' => [
+        'method' => 'GET',
+        'url' => 'https://www.thewebhatesme.com/health.json',
+        'headers' => [
+            'Content-Type' => 'application/json',
+        ],
+    ],
+    'drupal' => [
```

Comment:
key should not be the cms to be able to have different endpoints for the same cms.